tests/core/numerics/interpolator/test_main.cpp

```diff
@@ -675,11 +688,14 @@ TYPED_TEST_SUITE_P(ACollectionOfParticles_1d);
 
 TYPED_TEST_P(ACollectionOfParticles_1d, DepositCorrectlyTheirWeight_1d)
 {
-    // node 25 because assume 5 ghosts
-    EXPECT_DOUBLE_EQ(this->rho(25), 1.0);
-    EXPECT_DOUBLE_EQ(this->vx(25), 2.0);
-    EXPECT_DOUBLE_EQ(this->vy(25), -1.0);
-    EXPECT_DOUBLE_EQ(this->vz(25), 1.0);
+    constexpr auto interp = TypeParam::interp_order;
+
+    auto idx = 20 + this->layout.nbrGhosts(QtyCentering::dual);
```

Comment:
20 ? 
also why dual ?

Reply:
> 20 ?

20 = 25 - 5

Reply:
should this file be updated to also include dual tests in this PR?